packages/framework/src/Command/CheckRedisCommand.php

```diff
@@ -2,30 +2,45 @@
 
 namespace Shopsys\FrameworkBundle\Command;
 
+use Redis;
+use Shopsys\FrameworkBundle\Component\Redis\RedisFacade;
 use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Webmozart\Assert\Assert;
 
 class CheckRedisCommand extends Command
 {
+    protected const RETURN_CODE_OK = 0;
+    protected const RETURN_CODE_ERROR = 1;
+
     /**
      * @var string
      */
     protected static $defaultName = 'shopsys:redis:check-availability';
 
     /**
-     * @var \Redis[]
+     * @var \Shopsys\FrameworkBundle\Component\Redis\RedisFacade|\Redis[]
      */
-    protected $cacheClients;
```

Comment:
removed protected property of non final class

Reply:
fixed in e7806e5f6